workspaces/Cargo.toml

```
@@ -23,6 +23,7 @@ serde = "1.0"
 serde_json = "1.0"
 tokio = { version = "1", features = ["full"] }
 tokio-retry = "0.3"
+tracing = "0.1.29"
```

```suggestion
tracing = "0.1"
```

Prob better for a lib like this to not have this as the minimum